### DISCUSSION TITLE
Error processing Cubit mesh

### DISCUSSION THREAD

**Discussion by Dan2997925** [mfem/mfem#3835] [Q&A]
I'm reading a 3D cubit mesh from disk. I get an error from mesh::GetQuadOrientation, it prints out
base = [ x x x x]
test = [x x x x]
and then it exits with mfem::mfem_error

So what does this mean?

**Comment by v-dobrev** [comment]
It means that `test` is not a valid permutation of `base` -- e.g. for a quad it is not valid to just switch the first two indices.

**Reply by v-dobrev** [reply]
If you have a stack trace of which method called `Mesh::GetQuadOrientation` that may help narrow down the specific issue.
